### ISSUE TITLE
SPIKE - Enable Additional DB Sync Configuration/Optimization flavors

### ISSUE THREAD

**Issue by marshada** [input-output-hk/cardano-db-sync#1297]
Identify additional config arguments (flavors)

“no ledger state” option
—disable-ledger
—diesable-cache
—disable-epoch (reduced mem to 1GB from 24GB)

**Comment by erikd** [comment]
> —disable-epoch (reduced mem to 1GB from 24GB)

I am incredibly skeptical of that that flag alone can so drastically reduce memory consumption.

I would totally believe it for `--disable-ledger` but not that one.

**Comment by kderme** [comment]
Yes that's probably for --disable-ledger and it's from 12 to 1G.

**Comment by marshada** [comment]
OK I corrected it above

**Comment by marshada** [comment]
just realized this is a duplicate issue of #1195 , so am closing it.
